q3vcftools/src/au/edu/qimr/vcftools/Classify.java

```diff
@@ -162,7 +162,7 @@ private void writeOutput(List<ChrPositionRefAlt> recs, String output) throws Fil
 			ps.println(header);
 			for (ChrPositionRefAlt cp : recs) {
 //				logger.info("writing out: " + cp.getName());
-				ps.println(cp.getChromosome() + Constants.TAB + cp.getStartPosition() + "\t.\t" + cp.getName() + Constants.TAB + cp.getAlt() + "\t.\t.\t.");
+				ps.println(cp.getChromosome() + Constants.TAB + cp.getStartPosition() + "\t.\t" + cp.getRef() + Constants.TAB + cp.getAlt() + "\t.\t.\t.");
```

Comment:
While we're here, is this an opportunity to generate output in a nicer (more explicit) way, given that these look basically like VCF records? Essentially the same concatenation is used a couple more times below too.

Reply:
added a method to ChrPositionUtils that takes a ChrPosition (and some other bits of info) and returns a Vcf string